schemas/examples/basic/current.json

```
@@ -0,0 +1,47 @@
+{
+  "title": "examples/basic",
+  "description": "A basic schema not using references",
+  "$id": "/examples/basic/1.0.0",
+  "$comment": "status-draft",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "comment"
+  ],
+  "properties": {
+    "comment": {
+      "description": "A string property added in basic 1.0.0",
+      "type": "string"
```

```suggestion
      "title": "A comment",
      "description": "A string property added in basic 1.0.0",
      "type": "string"
```

Can we add titles to all of the examples? They are used by the Sample Service GUI so it will be good to encourage users to fill them out.

---

On referenced schema I've noticed the "title" and "description" gets copied across. Adding in advanced schema means the build replaces the values from the basic scheme. Getting SRC to pass required at least two builds when using build-all command.